gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/application/use_case/ImportApplicationCRDUseCase.java

```diff
@@ -150,6 +179,76 @@ private void createOrUpdateApplicationMetadata(Input input, BaseApplicationEntit
         }
     }
 
+    private void syncApplicationMembers(Input input, BaseApplicationEntity application) {
+        List<String> existingApplicationMembers = memberQueryService
```

Comment:
same remark about var, but I can see that there's a lot to update, so I'll make that one a global statement.